common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -117,10 +74,62 @@
             renderElements: function () {
                 var languages = this.state.config.transcriptLanguages;
 
+                var langTemplate = [
```

Comment:
I needed to be able to access variables, which wasn't possible the other way. I spoke with Andy A about this change and he was OK with it.